default.ct.yaml

```diff
@@ -7,4 +7,4 @@ chart-repos:
 - banzaicloud-stable=https://kubernetes-charts.banzaicloud.com
 excluded-charts:
 - lagoon-test
-helm-extra-args: --timeout 900s
+helm-extra-args: --timeout 1800s
```

Comment:
Did this help? This file is only to run the super basic connectivity tests on the charts other than `lagoon-test`, were they taking over 15min?

Reply:
removed this now